### PR TITLE
refactor: drop `lodash.trim` dependency

### PR DESCRIPTION
**Pull request by XhmikosR**
Supersedes #742 

Refs #598

### CODE REVIEW

**Comment by XhmikosR** [comment]
@Kreeg this seems to work fine from my tests. Does it LGTY?

File: lib/svg-sprite.js [hunk 1]
```diff
@@ -84,15 +83,16 @@ SVGSpriter.prototype._namespacePow = [];
...
 
         // If the name part of the file path is absolute
         if (name && path.isAbsolute(name)) {
             error = format('SVGSpriter.add: "%s" is not a valid relative file name', name);
         } else {
-            name = trimStart(trim(name), `${path.sep}.`) || path.basename(file);
-            svg = trim(svg);
+            name = trimStart(name.trim(), `${path.sep}.`) || path.basename(file);
+            // TODO: Avoid Buffer -> String -> Buffer conversion of svg.
+            svg = svg.toString().trim();
```
Comment (XhmikosR) [inline lib/svg-sprite.js:95]: Just wondering, would it make more sense to check if `svg` is Buffer with `Buffer.isBuffer()` and if so only then call `toString()`?

**Comment by XhmikosR** [comment]
After a closer look, this is not identical to what we had before. Previously, if you passed `null` or `undefined`, it would proceed silently. Now we'll get a TypeError:

```
C:\Users\xmr\Desktop\svg-sprite\lib\svg-sprite.js:86
        file = file.trim();
                    ^

TypeError: Cannot read properties of undefined (reading 'trim')
    at SVGSpriter.add (C:\Users\xmr\Desktop\svg-sprite\lib\svg-sprite.js:86:21)
    at C:\Users\xmr\Desktop\svg-sprite\example.js:62:17
    at Array.forEach (<anonymous>)
    at addFixtureFiles (C:\Users\xmr\Desktop\svg-sprite\example.js:61:11)
    at Object.<anonymous> (C:\Users\xmr\Desktop\svg-sprite\example.js:71:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)

Node.js v18.12.1
```

File: lib/svg-sprite.js (hunk 1 above)
Comment (Kreeg) [reply lib/svg-sprite.js]: This would seem a little overengineering for me.

**Review by Kreeg — APPROVED** [review]

**Pull request merged**